src/scss/components/checkbox/_index.scss

```diff
@@ -1,6 +1,7 @@
-@import (reference) 'vars';
-@import (reference) 'mixins';
-@import (reference) 'components/form/index';
+@import "vars";
```

Comment:
Remove import.

* "vars" is already imported in `_config.scss`

Reply:
Removed:

```css
@import "vars";
```